core/src/main/scala/quasar/fs/PathError.scala

```diff
@@ -29,11 +29,11 @@ sealed trait PathError2 {
 }
 
 object PathError2 {
-  private final case class PathExists0(path: AbsPath[Sandboxed])
+  final case class PathExists0(path: AbsPath[Sandboxed])
```

Comment:
:-1: These are intentionally private, use `PathError2#fold` or the prisms instead.